.gitattributes

```diff
@@ -0,0 +1,6 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
+*.bat           text eol=crlf
+
```

Comment:
Do we need this file? if not, shall we remove it

Reply:
We don't need, I will remove

Reply:
Fixed with [`07c225d`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/b7174d84783511ddefba62f2865291a99fb0717d)